packages/tools/src/tools/annotation/PlanarFreehandROITool.ts

```diff
@@ -743,11 +744,16 @@ class PlanarFreehandROITool extends AnnotationTool {
         continue;
       }
 
-      const { imageData, metadata } = image;
-      const canvasCoordinates = points.map((p) => viewport.worldToCanvas(p));
+      const { imageData, metadata, hasPixelSpacing } = image;
+      const imageId = viewport.getCurrentImageId();
```

Comment:
this would not work for recontructed planes (non-acquisition) since they don't have currentImageId

Reply:
Ah I see, I can change it to use canvas coordinates in that case